templates/taxi/driver_list.html

```diff
@@ -1,7 +1,7 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Driver List
+    <h1>Driver List <a href="forms.py">+</a>
```

Comment:
The link for adding a new driver is incorrectly set to `forms.py`. It should use the `{% url "taxi:driver-create" %}` tag to point to the correct URL for creating a new driver.